eng/Build.ps1

```diff
@@ -278,7 +274,7 @@ function Update-Arguments() {
 }
 
 
-function BuildSolution([string] $solutionName, $nopack) {
```

Comment:
This logically inverts the switch, that does not feel right.

Reply:
The switch was never nopack it was always the opposite. Check the condition where it is used.

Reply:
You are correct, my apologies.